docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -30,6 +30,16 @@ Follow the steps below to run the [TPC-C workload](https://github.com/yugabyte/t
 
 All the nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the DB cluster. Each test was run for `30 minutes` after the loading of the data.
```

Comment:
I know this paragraph isn't yours, but there's no need for the code font for "30 minutes" :)
```suggestion
All the nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the DB cluster. Each test was run for 30 minutes after the loading of the data.
```